api/lib/infrastructure/converter/snapshots-csv-converter.js

```diff
@@ -0,0 +1,65 @@
+const moment = require('moment');
+
+const headersWithoutCompetences = ['Nom', 'Prénom', 'Numéro Etudiant', 'Code Campagne', 'Date', 'Score Pix', 'Tests Réalisés'];
+
+module.exports = {
```

Comment:
remarque générale: le CSV c'est pas un format simple, je suis surpris qu'on gère ça sans bibliothèque idoine mais why not

Reply:
Ce qu'on aurait fait en majorité ici aurait du être fait dans tout les cas. 
La plupart des librairies partent d'un tableau déjà fait pour faire le CSV.
De mon point de vue, on peut l'ajouter, mais la logique dernière ne changerait pas beaucoup.